.gitignore

```diff
@@ -12,3 +12,4 @@
 *.env
 *.env_*
 !/tests/fixes/fixtures/*.env
+!/benches/fixtures/*.env
```

Comment:
Why these changes is needed?

Reply:
I noticed the `benches/fixtures/simple.env` as part of the original check benchmark function so I added an `benches/fixtures/compare.env`. I needed to commit it but it was blocked by `.gitignore`.